app/views/users/index.json.jbuilder

```diff
@@ -1,3 +0,0 @@
-# frozen_string_literal: true
```

Comment:
These jbuilder views aren't used as far as I can tell. I suspect the don't even work.